rskj-core/src/main/java/org/ethereum/net/client/ConfigCapabilitiesImpl.java

```diff
@@ -29,26 +29,29 @@
 import java.util.TreeSet;
 
 import static org.ethereum.net.client.Capability.RSK;
+import static org.ethereum.net.client.Capability.SNAP;
 import static org.ethereum.net.eth.EthVersion.fromCode;
 
 /**
  * Created by Anton Nashatyrev on 13.10.2015.
  */
 public class ConfigCapabilitiesImpl implements ConfigCapabilities{
 
-    private final SystemProperties config;
+    private final RskSystemProperties config;
 
     private SortedSet<Capability> allCaps = new TreeSet<>();
 
-    public ConfigCapabilitiesImpl(SystemProperties config) {
+    public ConfigCapabilitiesImpl(RskSystemProperties config) {
         if (config.syncVersion() != null) {
-            EthVersion eth = fromCode(config.syncVersion());
+            EthVersion eth =fromCode(config.syncVersion());
             if (eth != null) {
                 allCaps.add(new Capability(RSK, eth.getCode()));
+                if (config.isServerSnapshotSyncEnabled()) allCaps.add(new Capability(SNAP, eth.getCode()));
```

Comment:
Please use if with '{ }'. Why are we using `eth.getCode()` for snap aswell?  Maybe it is worth to have Snap protocol version aswell, just in case we modify something in the future.

Reply:
solved, SNAP capability now uses version `(byte) 1`